server/plugin/webhook.go

```diff
@@ -829,6 +817,18 @@ func (p *Plugin) postIssueCommentEvent(event *github.IssueCommentEvent) {
 			continue
 		}
 
+		post := &model.Post{
+			UserId: p.BotUserID,
+			Type:   "custom_git_comment",
+		}
+
+		repoName := strings.ToLower(repo.GetFullName())
+		commentID := event.GetComment().GetID()
```

Comment:
I think we can move this out of the loop

Reply:
1/5 that getting the commentID and assigning it as a prop should stay together as they semantically belong together.

Reply:
@ayusht2810 Note that these methods being called are not performing network requests. They are essentially helper methods that are accessing fields on the struct for us. @hanzei Sure we can keep it as is. `strings.ToLower` is happening each time but that's the only concern I could potentially have. The repo name is always going to be short.

Reply:
@mickmister Sure, that's fine with me.